src/Symfony/Component/Finder/Iterator/VcsIgnoredFilterIterator.php

```diff
@@ -27,7 +27,8 @@ final class VcsIgnoredFilterIterator extends \FilterIterator
 
     public function __construct(\Iterator $iterator, string $baseDir)
     {
-        $this->baseDir = $baseDir;
+        $escapedBaseDir = preg_quote($this->normalizePath($baseDir), '~');
+        $this->baseDirRegexp = "~^{$escapedBaseDir}~";
```

Comment:
why don't we use `0 === strpos()` instead?

Reply:
Done.